File: Python/1165.py
cont = int(input())

for i in range(0,cont):
    num = int(input())
    div = 0
    x=1

    while x <= num:
        if num % x == 0:
            div = div + 1
        x = x + 1
    
    if div > 2:
        print('{} nao eh primo'.format(num))
    
    else:
        print('{} eh primo'.format(num))

File: JavaScript/1165.js
var input = require('fs').readFileSync('dev/stdin', 'utf8');
var lines = input.split('\n');

var cont = parseInt(lines.shift()); 

for (var i = 0; i < cont ; i++) {
  var num = parseInt(lines.shift()); 
  var div = 0;
  var x=1;

  while (x <= num) {
    if (num % x == 0){
      div = div + 1
    }
    x = x + 1
  }

  if (div > 2){
    console.log(num + " nao eh primo")
  }
  else {
    console.log(num + " eh primo")
  }
}